docs/how-to-guides/software-management/software-support.md

```diff
@@ -1,34 +1,57 @@
 
-After setting up the sprint schedule, the technical lead should create a software support board and communicate the schedule to contributors.
+After setting up the sprint schedule, we create a software support board and communicate the schedule to contributors. The prioritization meeting should be scheduled some day the week before the support sprint. 
 
-## Step 2: Setting up the Project Board
+### Step 2: Setting up the Project Board
 Immediately after scheduling the support sprint, the technical lead should create a project board to facilitate the tracking, prioritization, and assignment of issues.  This has traditionally taken the form of a GitHub [projects board](https://github.com/orgs/DOI-USGS/teams/astrogeology-developers/projects) or a GitHub [discussion post](https://github.com/DOI-USGS/ISIS3/discussions), both of which provide automated tracking for issues that are hosted on GitHub. 
 
-!!! Note "Due to recent changes in permissions and organizational boundaries, project boards can only be created as private (within-organization), which precludes external contributors from viewing or editing the board."
 
-## Step 3: Notifying the Community
-With a project board in place, there is now a mechanism to store incoming issues, and the technical lead can notify the community of the upcoming sprint.  There is no "correct" way to notify community members, but the recommended method is via an MS Teams calendar event, which allows the host to send an email with an associated meeting invitation.
+### Step 3: Notifying the Community
+
+The community is notified via an email sent to anyone [subscribed to the newsletter](https://public.govdelivery.com/accounts/USDOIGS/signup/39118). 
 
 This email should include:
 
 1. The time of the prioritization meeting
+1. Link to the meeting, a new meeting should be created every time via MS Teams.
 1. The purpose of the meeting
 1. The start / end date of the sprint
 1. A solicitation for issues that are significant to the community
 1. Instructions on getting issues added to the board.
 
-
 ??? Tip "Optional Boilerplate Email for MS Teams Calendar Event"
     This meeting will be used to prioritize the issues found at < link to project board > for the upcoming support sprint scheduled < date range of support >.  The list of issues found on the project board is currently incomplete.  If the list doesn't contain work that you would like to see scheduled during this sprint, then feel free to add items to the list.  For contributors outside the ASC, please note that organizational policy does not currently allow read/write access to internal project boards.  If you'd like work to be scheduled during this sprint, please ensure that an issue has been created and email me at < your email > to get the work scheduled.
 
+??? Danger "For USGS devs: What mechanism do we use to send these emails out?"
```

Comment:
Is this a public doc? Why does it have a question in it for USGS developers?

Reply:
Again, this is kind of for both. I reference these docs from time to time. IDK why it being public disqualifies it from being useful to us as well.

Reply:
It doesn't disqualify. It is weird for a reader though. Image reading your banking statement or a cookbook and seeing intermittent notes to the authors. It hard breaks the flow.